userstorage/options.py

```diff
@@ -0,0 +1,14 @@
+# SPDX-FileCopyrightText: Nir Soffer <nirsof@gmail.com>
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+from . import errors
+
+
+def missing_handler(msg):
```

Comment:
This is never used since we use the copy in the package namespace. Keeping this is a trap for developers. If you do:

    from . import options
    options.missing_handler(msg)

You just ignored the user value.

The best way to avoid this issue is to not add this module, and keep the function in the package namespace (__init__.py).